src/Optimizely/Event/Builder/EventBuilder.php

```diff
@@ -23,6 +23,9 @@
 use Optimizely\ProjectConfig;
 use Optimizely\Utils\EventTagUtils;
 
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\Exception\UnsatisfiedDependencyException;
+
```

Comment:
What's the use of above 2 lines?

Reply:
So that the EventBuilder class, can instantiate UUID module object